python/paddle/v2/fluid/tests/book/test_label_semantic_roles.py

```diff
@@ -155,9 +155,18 @@ def main():
             name='crfw', learning_rate=mix_hidden_lr))
     avg_cost = fluid.layers.mean(x=crf_cost)
 
+    global_step = fluid.layers.create_global_var(
+        shape=[1], value=0, dtype='float32')
     # TODO(qiao)
     # check other optimizers and check why out will be NAN
-    sgd_optimizer = fluid.optimizer.SGD(learning_rate=0.0001)
+    sgd_optimizer = fluid.optimizer.SGD(
+        learning_rate=fluid.learning_rate_decay.exponential_decay(
```

Comment:
Will the old syntax also work? For example, will this line work with the new code `sgd_optimizer = fluid.optimizer.SGD(learning_rate=0.0001)`

Reply:
yes, it will work, _create_global_learning_rate will handle it if learning_rate is a float value